index.html

```diff
@@ -0,0 +1,265 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav class="navbar">
+            <a href="">
+                <h2>My Logo</h2>
+            </a>
+            <img src="images/hamburger.svg" alt="menu button">
+        </nav>
+    </header>
+    <section id="headline">
+        <h1>I’m Thinus Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-links">
+            <li>
+                <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/github.svg" alt="Github"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+            </li>
+        </ul>
+    </section>
+
+    <section id="works">
+        <div id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio.svg" alt="SnapshootPortfolio picture">
+            <div id="left_block">
+                <div id="primary_text">
+                    <h4 id="project_title">Tonic</h4>
+                    <div id="frame2">
+                        <div id="client">
+                            <h2 id="pcanopy">CANOPY</h2>
+                        </div>
+                        <img id="counter">
+                        <div id="role">
+                            <h3 id="pcareer">Back End Dev</h3>
+                        </div>
+                        <img id="counter">
+                        <div id="year_label">
+                            <h5 id="pcareer">2015</h5>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] Great job removing the sections tags from the lists 👏  , but as mentioned by both  the previous reviewer and the Sr Reviewer, we should be semantically correct and use the tags `<ul>` & `<li>` for lists, here's a hint on how to do it : 

❌ Incorrect : 
```HTML
       <div id="frame2">
                        <div id="client">
                            <h2 id="pcanopy">CANOPY</h2>
                        </div>
                        <img id="counter">
                        <div id="role">
                            <h3 id="pcareer">Back End Dev</h3>
                        </div>
                        <img id="counter">
                        <div id="year_label">
                            <h5 id="pcareer">2015</h5>
                        </div>
             </div>
```

✔️  Correct : 

```HTML
       <ul id="frame2">
                        <li id="client">
                            <h2 id="pcanopy">CANOPY</h2>
                        </li>
                        <li id="role">
                            <img id="counter">
                            <h3 id="pcareer">Back End Dev</h3>
                        </li>
                        <li id="year_label">
                            <img id="counter">
                            <h5 id="pcareer">2015</h5>
                        </li>
          </ul>
```

  _**🛑 Kindly fix it for all of your cards 🙏**_